src/aasthaJJun20/SumOfAllCharsInStringUsingAscii.java

```diff
@@ -0,0 +1,24 @@
+package aasthaJJun20;
+/*Assignment 28 : 16th July 2020
+
+WAP to sum all the numbers in given string, without using Character class method. 
+(Note : Use Ascii value concept) .
+
+Input: J7u9y8h1h8j4b7j3j6
+Output: Sum of all numbers in String is - 53.*/
+
+public class SumOfAllCharsInStringUsingAscii {
+	void findSumOfAllNumbers(String input){
+		int sum = 0;
+		for(int index = 0; index < input.length(); index++){
+			if(input.charAt(index) >= '0' && input.charAt(index) <= '9'){
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
sum += (str.charAt(index) - '0'); You can use this as well. 
Or  sum = sum + Character.getNumericValue(str.charAt(index);